feature/platform/tests/optics_power_and_bias_current_test/README.md

```diff
@@ -1,49 +1,102 @@
-# gNMI-1.13: Telemetry: Optics Power and Bias Current
+# gNMI-1.13: Optics Telemetry, Instant, threshold, and miscellaneous static info
 
 ## Summary
 
-Validate optics input power, output power and bias current.
+Validate optics related streaming telemetry performance monitoring parameters
+like input power, output power, bias current and so on.
+
+## Setup
+Optics test requirements alongside the platform FNT require 2 optics DUT
```

Comment:
what is FNT? Perhaps we can define that here?

Reply:
updated to functional tests